src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do List - Jose RM</title>
+  <!-- CSS style  -->
+  <link rel="stylesheet" href="./style.css">
+</head>
```

Comment:
- [ ] Your styling isn't looking great from my end and i think probably the way you have imported the CSS file in your HTML makes the styling got broken on my end. Kindly work on the styling and be sure it looks like the sneak peak you were given.

this is what the styling looks like at my end 👇 

<img width="424" alt="image" src="https://user-images.githubusercontent.com/78634841/236051346-848c20d3-d941-4e27-8ce6-ccde79c1118b.png">

Reply:
ooh !!!, ok, so sorry, I will fix the issue,